doajtest/unit/api_tests/test_apiv3_bulk_aritcles.py

```diff
@@ -0,0 +1,39 @@
+import json
+
+from doajtest.fixtures import article_doajxml
+from doajtest.fixtures.article_doajxml import DoajXmlArticleFixtureFactory
+from doajtest.helpers import DoajTestCase
+from portality import models
+from portality.util import url_for
+
+
+def load_json_by_handle(handle) -> str:
+    articles = [article.data for article in article_doajxml.to_articles(handle)]
+    data_json_str = json.dumps(articles)
+    return data_json_str
+
+
+def create_user_with_api_key():
+    acc = models.Account.make_account(email="a1@example.com", roles=["user", "api"])
+    acc.save(blocking=True)
+    return acc
+
+
+class TestBulkArticles(DoajTestCase):
+
+    def test_bulk_article_create_async__201(self):
```

Comment:
yes, new api will return 201 in mostly case, even invalid format in incoming article json.
all validation will be done on background job to speed response time of api to avoid timeout

in following or above examples, should be complete the background job without fail
```
curl --location 'http://localhost:5004/api/bulk/articles/async?api_key=eb22d3bad3c246869426b9cb6d2b4737' \
--header 'Content-Type: application/json' \
--data '[{"id": "abcdefghijk_article", "admin": {"in_doaj": true, "seal": false, "publisher_record_id": "some_identifier", "upload_id": "zyxwv
utsrqpo_upload_id"}, "bibjson": {"title": "Article Title", "identifier": [{"type": "doi", "id": "10.0000/SOME.IDENTIFIER.1"}, {"type": "pissn"
, "id": "1234-5678"}, {"type": "eissn", "id": "9876-5432"}], "journal": {"volume": "1", "number": "99", "publisher": "The Publisher", "title":
 "The Title", "language": ["EN", "FR"], "country": "US", "start_page": "3", "end_page": "21"}, "year": "1991", "month": "January", "link": [{"
url": "http://www.example.com/article_1", "type": "fulltext", "content_type": "HTML"}], "abstract": "This is the abstract for the example arti
cle. It can be quite a long field so there'\''s a fairly sizable chunk of text here. The weather today is grey, because I am writing this from
 Scotland, and the time is early evening. A cup of tea helps work'\''s pleasantness - in particular, a rooibos blend with vanilla and spice. I
t'\''s a tea that goes fairly well with ginger biscuits. Never dunked, of course. Rooibos is from South Africa; it'\''s a flavoursome caffeine
-free tea, ideal for the working day. I like mine fairly strong - steeping for 3-4 minutes gives this tea enough time to release its distincti
ve flavour, but without overpowering the delicate vanilla. ", "author": [{"name": "The Author", "affiliation": "University Cottage Labs", "orc
id_id": "https://orcid.org/0000-0001-1234-1234"}], "keywords": ["word", "key"], "subject": [{"scheme": "LCC", "term": "Economic theory. Demogr
aphy", "code": "HB1-3840"}, {"scheme": "LCC", "term": "Social Sciences", "code": "H"}]}, "created_date": "2000-01-01T00:00:00Z"}, {"id": "abcd
efghijk_article", "admin": {"in_doaj": true, "seal": false, "publisher_record_id": "some_identifier", "upload_id": "zyxwvutsrqpo_upload_id"},
"bibjson": {"title": "Article Title", "identifier": [{"type": "doi", "id": "10.0000/SOME.IDENTIFIER.2"}, {"type": "pissn", "id": "1234-5678"},
 {"type": "eissn", "id": "9876-5432"}], "journal": {"volume": "1", "number": "99", "publisher": "The Publisher", "title": "The Title", "langua
ge": ["EN", "FR"], "country": "US", "start_page": "3", "end_page": "21"}, "year": "1991", "month": "January", "link": [{"url": "http://www.exa
mple.com/article_2", "type": "fulltext", "content_type": "HTML"}], "abstract": "This is the abstract for the example article. It can be quite
a long field so there'\''s a fairly sizable chunk of text here. The weather today is grey, because I am writing this from Scotland, and the ti
me is early evening. A cup of tea helps work'\''s pleasantness - in particular, a rooibos blend with vanilla and spice. It'\''s a tea that goe
s fairly well with ginger biscuits. Never dunked, of course. Rooibos is from South Africa; it'\''s a flavoursome caffeine-free tea, ideal for
the working day. I like mine fairly strong - steeping for 3-4 minutes gives this tea enough time to release its distinctive flavour, but witho
ut overpowering the delicate vanilla. ", "author": [{"name": "The Author", "affiliation": "University Cottage Labs", "orcid_id": "https://orci
d.org/0000-0001-1234-1234"}], "keywords": ["word", "key"], "subject": [{"scheme": "LCC", "term": "Economic theory. Demography", "code": "HB1-3
840"}, {"scheme": "LCC", "term": "Social Sciences", "code": "H"}]}, "created_date": "2000-01-01T00:00:00Z"}]'
```